source/includes/steps-connect-tableau-mongosql-plugin.yaml

```diff
@@ -1,80 +1,163 @@
 title: Download the C Authentication Plugin library.
 ref: download-plugin
 level: 4
-pre: |
-  Navigate to the `releases page
-  <https://github.com/mongodb/mongosql-auth-c/releases>`_
-  and download the ``mongosql_auth`` plugin library. Save
-  the file ``mongosql_auth.so`` to the
-  ``/usr/local/mysql/lib/plugin/`` directory.
+stepnum: 1
+content: | 
+
+  Navigate to the `releases page <https://github.com/mongodb/mongosql-auth-c/releases>`_ 
+  then download the ``mongosql_auth`` plugin library.
+
+  .. list-table::
+     :widths: 20 80
+     :stub-columns: 1
+
+     * - macOS
+       - Save the library file ``mongosql_auth.so`` to the 
+         ``/usr/local/mysql/lib/plugin/`` directory.
+
+         If this directory does not exist, create it:
+
+         .. class:: copyable-code
+         .. code-block:: sh
+
+            sudo mkdir -p /usr/local/mysql/lib/plugin -m 755
+            sudo chown -R {user}:admin /usr/local/mysql
+            tar -xvf /Users/{user}/Downloads/mongosql-auth-osx-x86_64-v{version}.tgz
+            cp /Users/{user}/Downloads/mongosql-auth-osx-x86_64/lib/mongosql_auth.so ./
+
```

Comment:
This is something we do with Cloud docs quite a bit: being explicit. I added an example frame around it to make clear this is suggested or possible, but not verbatim.